gdal/doc/source/drivers/vector/mssqlspatial.rst

```diff
@@ -212,11 +212,15 @@ Creating a layer from an OGR data source
 
       ogr2ogr -overwrite -f MSSQLSpatial "MSSQL:server=.\MSSQLSERVER2008;database=geodb;trusted_connection=yes" "rivers.tab"
 
+      ogr2ogr -overwrite -f MSSQLSpatial 'MSSQL:server=127.0.0.1;database=TestDB;UID=SA;PWD=DummyPassw0rd' "rivers.gpkg"
```

Comment:
If this is portable example, it may be better to consider peace of mind of Windows users and replace `'` with `"` :)